### PR TITLE
chore: qa for Ledger custom tokens

### PR DESCRIPTION
**Pull request by r4mmer**
### Acceptance Criteria

Update QA with new interactions for custom tokens on hardware wallet with Ledger.

### Security Checklist
- [ ] Make sure you do not include new dependencies in the project unless strictly necessary and do not include dev-dependencies as production ones. More dependencies increase the possibility of one of them being hijacked and affecting us.

### CODE REVIEW

File: QA.md [hunk 1]
```diff
@@ -201,19 +201,45 @@ You can connect your wallet to the testnet (https://node1.foxtrot.testnet.hathor
...
+    1. Go to the custom token wallet page and confirm that the key icon is showing next to the name (and not a checkmark).
+    1. If there is a checkmark icon, go to settings and untrust all tokens.
+    1. Click on 'Send tokens' on the top menu again.
+    1. Change the token dropdown to the custom token and try to send any amount. A modal will be displayed saying there are unverified custom tokens and the list should have the custom token added to the transaction.
+    1. Go to the custom token wallet screen and sign the custom token with Ledger (key icon next to the name).
+    1. Go back to the 'Send tokens' screen and click on 'Add another token' button and send HTR to an address and the custom token to the second copied address. Make sure both amounts are less than the balance available. A modal will be displayed asking to confirm operation on Ledger.
+    1. On Ledger you should check that each token is being sent to the correct address then click both buttons on the 'Approve' screen.
+    1. After transaction is completed, it closes the modal and goes back to main wallet screen. The first transaction on the list must have value 0.00.
```
Comment (tuliomir) [inline QA.md:241]: The value will be shown as `0.00` only if the two output addresses selected on step 1 are from the same wallet.
If the addresses are for another wallet, the full value will be shown.

Maybe it's a good idea to specify on step 1 that the output addresses to be used here must be from the same wallet as the input.

File: QA.md [hunk 2]
```diff
@@ -201,19 +201,45 @@ You can connect your wallet to the testnet (https://node1.foxtrot.testnet.hathor
     1. Click on 'Generate new address' and validate it has changed.
     1. Confirm again address on the wallet and on Ledger (by clicking 'Show full address').
 
+1. Ledger custom token wallet screen
```
Comment (tuliomir) [inline QA.md:204]: Minor formatting change
```suggestion
1. **Ledger custom token wallet screen**
```

File: QA.md [hunk 3]
```diff
@@ -201,19 +201,45 @@ You can connect your wallet to the testnet (https://node1.foxtrot.testnet.hathor
...
+    1. Register the token and go to the token wallet screen.
+    1. Check that there is a key icon next to it's name and click the key icon.
+    1. A modal should appear, confirm that the token symbol, name and uid are correct.
+    1. Click the "Sign token" button and check the Ledger device, it should say "Confirm token data".
+    1. Confirm that the symbol, name and uid are correct (use buttons to scroll).
+    1. Click both buttons on the "Reject" step and check that an error message "User denied token on Ledger" appears on the wallet.
+    1. Try to sign the token again but this time click both buttons on the "Accept" step, a success message should show on the wallet.
+    1. After closing the modal, check that that the key icon is now an unclicable checkmark icon and hovering shows the message "Token signed with Ledger".
+
+1. Ledger reset token signature
```
Comment (tuliomir) [inline QA.md:215]: Minor formatting change
```suggestion
1. **Ledger reset token signature**
```

File: QA.md (hunk 1 above)
Comment (r4mmer) [reply QA.md:241]: Done in [e58aa9e](https://github.com/HathorNetwork/hathor-wallet/pull/278/commits/e58aa9ebd61ab0f3106e83000d7c1b28a36638ea)

**Review by tuliomir — APPROVED** [review]

File: QA.md [hunk 4]
```diff
@@ -201,19 +201,45 @@ You can connect your wallet to the testnet (https://node1.foxtrot.testnet.hathor
...
+    1. If there is a checkmark icon, go to settings and untrust all tokens.
+    1. Click on 'Send tokens' on the top menu again.
+    1. Change the token dropdown to the custom token and try to send any amount. A modal will be displayed saying there are unverified custom tokens and the list should have the custom token added to the transaction.
+    1. Go to the custom token wallet screen and sign the custom token with Ledger (key icon next to the name).
+    1. Go back to the 'Send tokens' screen and click on 'Add another token' button and send HTR to an address and the custom token to the second copied address. Make sure both amounts are less than the balance available. A modal will be displayed asking to confirm operation on Ledger.
+    1. On Ledger you should check that each token is being sent to the correct address then click both buttons on the 'Approve' screen.
+    1. After transaction is completed, it closes the modal and goes back to main wallet screen. The first transaction on the list must have value 0.00.
+    1. Click on the first transaction and confirm that it has 4 outputs. The extra 2 outputs are change outputs and should be marked with 'Your address'.
```
Comment (pedroferreira1) [inline QA.md:242]: After that I think we should close and open the wallet and guarantee that the load will load the signature correctly of the token from ledger.

Comment (r4mmer) [reply QA.md:242]: Done in [5fb6fc2](https://github.com/HathorNetwork/hathor-wallet/pull/278/commits/5fb6fc20658b32e10df86b608bd49ee9f5f18cf4)

**Review by pedroferreira1 — APPROVED** [review]

**Pull request merged**